lib/es-reindex.rb

```diff
@@ -52,37 +44,48 @@ def go!
     end
   end
 
+  def confirm
+    confirm_msg = "Confirm or hit Ctrl-c to abort...\n" 
+    printf "Copying '%s/%s' to '%s/%s'%s\n  #{confirm_msg}",
```

Comment:
I think we'll want this message output (the `Copying '%s/%s' to '%s/%s'%s ` part) whether or not it's from the CLI; it's useful feedback, even just in the capistrano output (where we'll eventually see all this from)...